travel_platform/smart_ordering.py

```diff
@@ -22,24 +22,41 @@ def smart_ordering(data_dict, filter_by, order_by='ASC', limit=100):
     Returns:
         list: Sorted list of dictionaries with applied filters.
     '''
+    order_by = order_by.upper()
+    # print filter_by, order_by, limit
     if order_by == 'DESC':
         sorted_data = sorted(data_dict,
                              key=lambda note: (filter_by not in note, note.get(filter_by)),
                              reverse=True)
     else:
         sorted_data = sorted(data_dict,
                              key=lambda note: (filter_by not in note, note.get(filter_by)))
+    if filter_by == None:
+        return sorted(data_dict)
 
     return sorted_data[:limit]
 
-print('Sorted by name:')
+'''
 for i in smart_ordering(data, 'name'):
     print i
+
 print('---'*15)
-print('Sorted by age:')
-for i in smart_ordering(data, 'age', 'DESC'):
+
+for i in smart_ordering(data, 'age', 'Desc', 5):
     print i
+
 print('---'*15)
-print('Sorted by gender:')
-for i in smart_ordering(data, 'gender', 'ASC', 3):
+
```

Comment:
you don't need this checks inside the module.
Please add some test if you see any reason in this